src/js/game/LevelMVC/LevelView.js

```diff
@@ -2309,9 +2309,10 @@ module.exports = class LevelView {
         xOffset = this.blocks[blockType][2];
         yOffset = this.blocks[blockType][3];
         sprite = group.create(xOffset + 40 * x, yOffset + group.yOffset + 40 * y, atlas, frame);
-        frameList = Phaser.Animation.generateFrameNames("Prismarine", 0, 5, "", 0);
-        sprite.animations.add("idle", frameList, 5, true);
-        this.playScaledSpeed(sprite.animations, "idle", 0.1);
+        const options = [1500, Phaser.Easing.Sinusoidal.InOut, true, 0, -1, true];
+        const tween = this.addResettableTween(sprite).to({
+          alpha: 0.5,
+        }, ...options);
```

Comment:
Is there a specific reason not to just write this as

```js
const tween = this.addResettableTween(sprite).to({
  alpha: 0.5,
}, 1500, Phaser.Easing.Sinusoidal.InOut, true, 0, -1, true);
```

Reply:
no, not specifically. i saw the convention elsewhere in the repo and thought it would be nice to clean up the options params a bit